dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/ws/crypto/util/PasswordCipherUtil.java

```diff
@@ -77,11 +77,13 @@ public class PasswordCipherUtil {
     private static final String CUSTOM_COLON = "custom:";
     private static final String XOR = "xor";
     private static final String AES = "aes";
+    private static final String AES_128 = "aes-128";
+    private static final String AES_256 = "aes-256";
     private static final String HASH = "hash";
 
     private static final byte XOR_MASK = 0x5F;
 
-    private static final String[] SUPPORTED_CRYPTO_ALGORITHMS_DEFAULT = new String[] { XOR, AES, HASH };
+    private static final String[] SUPPORTED_CRYPTO_ALGORITHMS_DEFAULT = new String[] { XOR, AES, AES_128, AES_256, HASH };
```

Comment:
i think we should remove the `AES_256` static variable since:
- `encipher_internal` only supports `AES` and `AES_128`
- the description in UtilityOptions.nlsprops only mentions aes and aes-128

Reply:
I am doing this intentionally for future support